bubble_sort_by.rb

```diff
@@ -0,0 +1,34 @@
+def bubble_sort_by(arr)
```

Comment:
I'm getting nill when testing this method.  
       expected: ["hi", "hey", "hello"]
           got: nil
     
       (compared using ==)

Also you need to right a case to check if no block is given it show raise and error!

Reply:
?????? hey I'm learning about ruby, do you can bem more friendly in your review. This keep more confused.